recipe/dashboard/api/userdetails/userEmailVerifyPut.go

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+*
+* This software is licensed under the Apache License, Version 2.0 (the
+* "License") as published by the Apache Software Foundation.
+*
+* You may not use this file except in compliance with the License. You may
+* obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+ */
+
+package userdetails
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/supertokens/supertokens-golang/recipe/dashboard/dashboardmodels"
+	"github.com/supertokens/supertokens-golang/recipe/emailverification"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+type userEmailVerifyPutResponse struct {
+	Status string `json:"status"`
+}
+
+type userEmailVerifyPutRequestBody struct {
+	UserID   string `json:"userId"`
+	Verified bool   `json:"verified"`
+}
+
+func UserEmailVerifyPut(apiInterface dashboardmodels.APIInterface, options dashboardmodels.APIOptions) (userEmailVerifyPutResponse, error) {
+	body, err := supertokens.ReadFromRequest(options.Req)
+
+	if err != nil {
+		return userEmailVerifyPutResponse{}, err
+	}
+
+	var readBody userEmailVerifyPutRequestBody
+	err = json.Unmarshal(body, &readBody)
```

Comment:
if `verified` is not part of the request body, the user email will be unverified. Is that expected behaviour?